models/cis.cattle.io.clusterscan.js

```diff
@@ -0,0 +1,40 @@
+import { CIS } from '@/config/types';
+import { downloadFile } from '@/utils/download';
+import { colorForState } from '@/plugins/steve/resource-instance';
+
+export default {
+  availableActions() {
+    const out = this._standardActions;
+    const downloadLogs = {
+      action:     'downloadReport',
+      enabled:    true,
+      icon:       'icon icon-fw icon-chevron-right',
+      label:      'Download Report',
+      total:      1,
+    };
+
+    out.push(downloadLogs);
+
+    return out;
+  },
+
+  downloadReport() {
+    return async() => {
+      const owned = await this.getOwned();
+      const reportCRD = owned.filter(each => each.type === CIS.REPORT)[0];
```

Comment:
Either way works but FYI `array.filter(...)[0]` is the same as  `array.find(...)`